content/Watsonx/WatsonxData/105.md

```diff
@@ -3,6 +3,7 @@ title: '105: Data Ingest'
 timeToComplete: 45
 updated: 2023-09-18
 ---
+<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />
```

Comment:
Unnecessary if there are no quiz alerts on this page?

Reply:
Removed